include/vcpkg/vcpkgpaths.h

```diff
@@ -60,6 +61,18 @@ namespace vcpkg
     struct PackageSpec;
     struct Triplet;
 
+    struct ContentAndHash
+    {
+        ContentAndHash(std::string&& content, Hash::Algorithm algo);
+
+        StringView content() const { return content_; }
+        StringView hash() const { return hash_; }
+
```

Comment:
I think these should return const string& since all the places are doing to_string making StringView a pessimization.